src/dxtbx/array_family/flex_table_suite.h

```diff
@@ -358,6 +431,21 @@ namespace dxtbx { namespace af { namespace flex_table_suite {
     return column.apply_visitor(visitor);
   }
 
+  template <typename T>
+  bool compare_columns(T &self, T &other) {
+    typedef typename T::const_iterator iterator;
+    DXTBX_ASSERT(self.nrows() == other.nrows());
```

Comment:
I contemplated `return false` here as they are not equal but I can see for debugging it is more useful to have the `ASSERT` - however comment for this to explain to future reader?

Reply:
`ValueError`?